### PR TITLE
Update dev container for Ethereum Relay && Clusters

### PR DESCRIPTION
**Pull request by eddyjlsh**
A few features here: 
- The devcontainer will now automatically start a cluster of substrate nodes for development/testing
- The project includes `mangata-{bridge | node | sdk}` as developer dependencies
- A simple cluster test has been added, checking six nodes remain in sync
- Ethereum dependencies for testing/development have been added
- An Eth relayer has been added for testing purposes
- Helper scripts have been included

Might help to demo this due to the high number of file changes, but existing functionality should be unaffected.

### CODE REVIEW

**Comment by goncer** [comment]
@eddyjlsh  it seems that there are some linter issues. Plz fix them otherwise, this PR can not be merged. :)

**Comment by goncer** [comment]
Maybe some updates in the readme about how to use with new stuff would be useful. what do you think?

File: .github/workflows/lint-test.yml [hunk 1]
```diff
@@ -41,7 +41,7 @@ jobs:
       - name: Install dependencies
         run: yarn
       - name: Tsc code quality check
-        run: yarn tsc-lint
+        run: yarn tsc-lint --skipLibCheck
```
Comment (goncer) [inline .github/workflows/lint-test.yml:44]: undo!

File: package.json [hunk 2]
```diff
@@ -35,6 +38,7 @@
     "test": "jest",
     "test-parallel": "rm nonce.db ; rm sudo.lock; jest --verbose --ci --group=parallel",
     "test-playground-pararell": "rm nonce.db ; rm sudo.lock; jest test/exploratory/parallelTesting/transfer/* --verbose",
+    "test-cluster": "npx ts-node ./test/cluster/index.ts",
```
Comment (goncer) [inline package.json:41]: do you mind instead of a index.tx do it in a jest test?

File: test/cluster/queries.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import { Node } from "./types";
```
Comment (goncer) [inline test/cluster/queries.ts:1]: moe this file to  /utils folder 
plz!

Comment (goncer) [reply test/cluster/queries.ts:1]: move**

File: .github/workflows/lint-test.yml (hunk 1 above)
Comment (goncer) [reply .github/workflows/lint-test.yml:44]: or  keep it ,  IDK :)

File: test/cluster/api.ts [hunk 4]
```diff
@@ -0,0 +1,185 @@
+import { ApiPromise, WsProvider } from "@polkadot/api";
```
Comment (goncer) [inline test/cluster/api.ts:1]: plz move this to utils folder

File: test/cluster/index.ts [hunk 5]
```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-console */
```
Comment (goncer) [inline test/cluster/index.ts:1]: plz, 
create a proper test instead of an index.tx :)

File: test/cluster/types.ts [hunk 6]
```diff
@@ -0,0 +1,10 @@
+import { ApiPromise } from "@polkadot/api";
```
Comment (goncer) [inline test/cluster/types.ts:1]: maybe move this to utils? 
I think multinode deserves an special subfoldering like UI tests :)

**Review by goncer — CHANGES_REQUESTED** [review]

**Comment by eddyjlsh** [comment]
I agree with all comments but I'm already working on a branch of this branch to add more cluster functionality so I'll merge this and make those changes in that branch.

**Comment by goncer** [comment]
sure! np

**Review by goncer — APPROVED** [review]

**Pull request merged**